README.md

```diff
@@ -144,3 +145,13 @@ require("lspconfig").djlsp.setup({
         root_dir = require("lspconfig.util").root_pattern("manage.py", ".git"),
 })
 ```
+
+If you want to access the log while developing, add the `--enable-log` flag to the cmd.
+The logfile will be written to a file in your current working directory named `djlsp.log`.
+
+``` lua
+require("lspconfig").djlsp.setup({
+        cmd = { "/path/to/django-template-lsp/.venv/bin/djlsp", "--enable-log" },
+})
+```
+
```

Comment:
I just figured this out while developing. Somehow I have not got any log with `:LspLog` in vim. There were only errors logged and lots of noice in it. Is that the intended way if using it?

Reply:
I think we need to use lsp log feature or maybe print logs strerr

Reply:
Ok, so do I understand this correct, using the `--enable-log` feature (like I described it) is currently the intended way but not the final solution?

Reply:
good question, maybe it can work like you did cache in PR. Default off when given without argument it will logs back to client running lsp and when given path it will log to that path.